src/constructors.jl

```diff
@@ -146,3 +146,38 @@ TaylorModelN(a::T, ord::Integer, x0::IntervalBox{N,T}, dom::IntervalBox{N,T}) wh
 @inline polynomial(tm::TaylorModelN) = tm.pol
 @inline domain(tm::TaylorModelN) = tm.dom
 @inline expansion_point(tm::TaylorModelN) = tm.x0
+
+
+"""
+    TMSol{N,T,V1,V2,M}
+
+Structure containing the solution of a validated integration.
+
+# Fields
+    `time :: AbstractVector{T}`  Vector containing the expansion time of the `TaylorModel1` solutions
+
+    `fp   :: AbstractVector{IntervalBox{N,T}}`  IntervalBox vector representing the flowpipe
+
+    `xTMv :: AbstractMatrix{TaylorModel1{TaylorN{T},T}}`  Matrix whose entry `xTMv[i,t]` represents 
+    the `TaylorModel1` of the i-th dependent variable, obtained at time time[t].
+"""
+struct TMSol{N,T<:Real,V1<:AbstractVector{T},V2<:AbstractVector{IntervalBox{N,T}},
+        M<:AbstractMatrix{TaylorModel1{TaylorN{T},T}}}
+    time :: V1
+    fp   :: V2
+    xTM  :: M
+
+    function TMSol(time::V1, fp::V2, xTM::M) where 
+            {N,T<:Real,V1<:AbstractVector{T},V2<:AbstractVector{IntervalBox{N,T}},
+            M<:AbstractMatrix{TaylorModel1{TaylorN{T},T}}}
+        @assert length(time) == length(fp) == size(xTM,2) && N == size(xTM,1)
+        return new{N,T,V1,V2,M}(time, fp, xTM)
+    end
+end
+
+get_time(a::TMSol) = getfield(a,:time)
```

Comment:
I would suggest to add a getter function for the normalized time domain of a `TMSol` at a given time. In ReachabilityAnalysis, such function is called `domain(R::TaylorModeReachSet)`, see [here](https://github.com/JuliaReach/ReachabilityAnalysis.jl/blob/master/src/ReachSets/TaylorModelReachSet.jl#L40))

Such helper function can be used in several places below. The implementation would be:

```julia
get_time_domain(a::TMsol, n::Int) = domain(getfield(a, :xTM)[1, n])
```

Reply:
Good point, thanks! Followed your suggestion, and implemented a `domain` function for `TMSol`s. As for the getter functions, I reused `expansion_point`, `flowpipe`, and `get_xTM`.